### ISSUE TITLE
Problems on building model : DCCRN on DNS-Challenge dataset

### ISSUE THREAD

**Issue by realkris** [asteroid-team/asteroid#446]
This project is really helpfull for noobs like me ! 
I want to make a DCCRN model(on DNS-Challenge official dataset) for class project as the noise cancellation module.
Here's What I've done now.
I modified the model.py script in egs/dns-challenge/baseline according to #278, but I have several questions that confused

1. In function distance(), where does the "F" come from?
2. how can I change the channels of DCCRN? I saw there's a "_dccrn_architectures.py " seems like a config of DCRNN under asteroid/masknn, but I didn't find where it is called in model architecture. 
3. Do we only have DCCRN-CL as the architecture to use? is that because DCCRN-CL is better than the "DCCRN - E/R/C"?

So basically I just modified the model.py scripy and dataset folder in **baseline**/conf.yml. Will that be enough? My English is not good, maybe I missed something?  or could you guys just give me an example.(I will train on the official DNS dataset) Thank you for your help!

**Comment by jonashaag** [comment]
I think it’s easiest to start from this recipe: https://github.com/asteroid-team/asteroid/tree/master/egs/librimix/DCCRNet

F is an alias convention for `torch.nn.functional`.

CL is the only variant available because nobody implemented the others yet.

What modification to the channels do you want to make? Input, output, or inner (ie. model size)?

**Comment by realkris** [comment]
Thank you for your reply! 
yes, model size, I want to reduce the size of model cause I want to run it on portable devices. is it correct to change the kernel_num from like [32, 64, 128, 256, 256, 256] to [16, 32, 64, 128, 256, 256]?

**Comment by jonashaag** [comment]
You’ll have to try. Nobody knows. But doesn’t look crazy wrong.

**Comment by realkris** [comment]
OK, thank you for your help, I'll continue trying it!

**Comment by wzx-cr** [comment]
你好，张博士，我能否借鉴一下，您在DNS数据集上的DCCRN模型的修改？
